go/cmd/vtbackup/vtbackup.go

```diff
@@ -202,6 +208,15 @@ func main() {
 		log.Errorf("Couldn't prune old backups: %v", err)
 		exit.Return(1)
 	}
+
+	if keepAliveTimeout > 0 {
```

Comment:
Added this for local testing, but I think this could also be useful in a K8s context to allow the next Prometheus scrape interval to pass before the process exits.